src/ipfsHttpClient.ts

```diff
@@ -1,9 +0,0 @@
-import * as ipfsHttpClient from 'ipfs-http-client'
-
-window.client = ipfsHttpClient.create({
-  host: 'ipfs.io',
-  port: 443,
-  protocol: 'https'
-})
-
-export { ipfsHttpClient }
```

Comment:
🔪 killed unnecessary code.

Reply:
While we may not want a window object, we could be exporting this client and using it in multiple checks/components. I don't think we want to remove modularity while killing global namespaces.

Reply:
Fixed, added a singleton for ipfsClient.